CHANGELOG.adoc

```diff
@@ -139,6 +139,7 @@ Thanks to this capability, a specifier can easily retrieve a variable overriden
 - https://github.com/eclipse-sirius/sirius-web/issues/3096[#3096] [tree] Tree representations can now support filters
 - https://github.com/eclipse-sirius/sirius-web/issues/3158[#3158] [diagram] Add a new tool to apply adjust-size on diagram node.
 - https://github.com/eclipse-sirius/sirius-web/issues/3201[#3201] [diagram] Add support for tools on multiple diagram elements selection (hide, fade and pin).
+- https://github.com/eclipse-sirius/sirius-web/issues/3236[#3236] [diagram] Add Is Collapsed By Default Expression in NodeDescription entity in the view DSL.
```

Comment:
That's not for `[diagram]`, that's for `[view]` but for the `[diagram]` entry you can specify that you have added a new predicate to specify if a node is collapsed by default.

Reply:
done